File: Rust/bee_1048.rs
use std::io::{self, Read};

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();

    let salario: f64 = input.trim().parse().unwrap();
    let novo_salario: f64;
    let reajuste: f64;
    let mut percentual: f64 = 0.0;

    if salario > 0.0 && salario <= 400.0 {
        percentual = 0.15;
    } else if salario > 400.0 && salario <= 800.0 {
        percentual = 0.12;
    } else if salario > 800.0 && salario <= 1200.0 {
        percentual = 0.10;
    } else if salario > 1200.0 && salario <= 2000.0 {
        percentual = 0.07;
    } else if salario > 2000.0 {
        percentual = 0.04;
    }

    reajuste = salario * percentual;
    novo_salario = salario + reajuste;

    println!(
        "Novo salario: {:.2}\nReajuste ganho: {:.2}\nEm percentual: {:.0} %",
        novo_salario,
        reajuste,
        (percentual * 100.0)
    );
}

File: Python/bee 1048.py
salario = float(input())

reajuste = 0

if salario >= 0 and salario <= 400:
    reajuste = 0.15
elif salario > 400 and salario <= 800:
    reajuste = 0.12
elif salario > 800 and salario <= 1200:
    reajuste = 0.10
elif salario > 1200 and salario <= 2000:
    reajuste = 0.07
else:
    reajuste = 0.04

print(f"Novo salario: {(salario + reajuste * salario):.2f}\nReajuste ganho: {(salario * reajuste):.2f}\nEm percentual: {(reajuste * 100):.0f} %")